src/pages/gd/Portfolio/index.tsx

```diff
@@ -82,6 +78,12 @@ const Portfolio = () => {
         }
     }, [account, mainnetChainId, fuseChainId])
 
+    const showNotice = data?.list.map(stake => {
+      return (stake.protocol === LIQUIDITY_PROTOCOL.GOODDAO && !stake.isV2
+              && stake.protocol === LIQUIDITY_PROTOCOL.GOODDAO && !stake.isV3
```

Comment:
liquidity protocol is the same here.

Reply:
should shownotice just be a boolean?
ie data?.list.find()
if we found one contract we show the message

Reply:
thats what this returns?

find would return value, here I return boolean[] | undefined
I don't see why I need the value of the stake?